graspologic/match/gmp.py

```diff
@@ -228,17 +232,33 @@ def fit(self, A, B, seeds_A=[], seeds_B=[], S=None):
             "maximize": self.gmp,
             "partial_match": partial_match,
             "S": S,
-            "rng": self.random_state,
             "P0": self.init,
             "shuffle_input": self.shuffle_input,
             "maxiter": self.max_iter,
             "tol": self.eps,
         }
-
-        res = min(
-            [quadratic_assignment(A, B, options=options) for i in range(self.n_init)],
-            key=lambda x: x.fun,
-        )
+        if self.n_init > 1:
```

Comment:
@asaadeldin11 I don't even think we need this if/else, thoughts?

Reply:
would make the code simpler is all

Reply:
i had it like that before but the tests were taking way slower

Reply:
i think creating a `Parallel` joblib object takes time

Reply:
interesting, I guess there is a bit of overhead to it. can leave as is for now, maybe add a comment?

Reply:
so not worth it if `n_init` = 1 i think

Reply:
you had `n_jobs` set to -1 tho? what about when `n_jobs` is default 1?

Reply:
regardless I guess I don't care too much but I thought they were smart enough about that stuff in joblib. maybe not

Reply:
i can try again with `n_jobs` set to 1